doc/manual/manual.tex

```diff
@@ -6656,7 +6656,71 @@ \subsubsection{Crustal deformation}
+
+\lstinputlisting[language=prmfile]{cookbooks/continental_extension/continental_extension_boundary_conditions.prm}
+
+Sections of the lithosphere with distinct properties are represented by compositional fields for the upper crust (20 km thick), lower crust (10 km thick) and mantle lithosphere (70 km thick).  A mechanically weak seed within the mantle lithosphere helps localize deformation.  Material (viscous flow law parameters, cohesion, internal friction angle) and thermodynamic properties for each compositional field are based largely on previous numerical studies.   Dislocation creep viscous flow parameters are taken from published deformation experiments for wet quartzite \cite{RB04}, wet anorthite \cite{RGWD06} and dry olivine \cite{HK04}. 
+
+\lstinputlisting[language=prmfile]{cookbooks/continental_extension/continental_extension_composition.prm}
+
+The initial thermal structure, radiogenic heating model and associated thermal properties are consistent with the prescribed thermal boundary conditions and produce a geotherm characteristic of the continental lithosphere.  The equations defining the initial geotherm \cite{Cha86} follow the form
+
+\begin{align}
+  \label{eq:continental-geotherm-1}
+  T(z) &= T_T + \frac{q_T}{k}z - \frac{Az^2}{2k}
+\end{align}
+
+where T is temperature, z is depth, $T_T$ is the temperature at the layer surface (top), $q_T$ is surface heat flux, $k$ is thermal conductivty, and $A$ is radiogenic heat production.
+
+For a layer thickness $\Delta z$, the basal temperature ($T_B$) and heat flux ($q_B$) are
+
+\begin{align}
+  \label{eq:continental-geotherm-2}
+  T_B &= T_T + \frac{q_T}{k} \Delta z - \frac{A \Delta z^2}{2k}
+  \\
+  \label{eq:continental-geotherm-3}
+  q_B &= q_T - A \Delta z
+\end{align}
+
+In this example, specifying the top (273 K) and bottom temperature (1573 K), thermal conducitvity of each layer and radiogenic heat production in each layer provides enough constraints to successively solve for the temperature and heat flux at the top of the lower crust and mantle.
+
+As noted above, the mechanically weak seed placed within the mantle localizes the majority deformation onto two conjugate shear bands that propogate from the surface of the seed to the free surface.  After 5 million years of extension background 'stretching' is clearly visible in the strain-rate field, but deformation is still largely focused within the set of conjugate shear bands originating at the weak seed  (\ref{fig:continental_extension_cookbook_strainrate_5e6yr}).  As expected, crustal thickness and surface topography patterns reveal a relatively symmetric horst and grabben structure, which arises from displacements along the shear bands (\ref{fig:continental_extension_cookbook_upper_5e6yr}).  
```

Comment:
enclose "stretching" into "backtick stretching forwardtick" to get the correct latex mark. (I can't write the sign for backtick here because it is a special markdown character, but you know what I meant.)